WordPress/Classes/ViewRelated/Stats/Shared Views/GrowAudienceCell.swift

```diff
@@ -143,18 +143,29 @@ class GrowAudienceCell: UITableViewCell, NibLoadable {
 
     private enum Strings {
 
-        static let viewsCountDescriptionSingular =
```

Comment:
If the diff in this this file is too noisy because of the formatting and indentation changes, checkout the isolated commit that adds the reverse-DNS keys [`9162d30` (#19635)](https://github.com/wordpress-mobile/WordPress-iOS/pull/19635/commits/9162d30220ccee1d643d2e72acaa47e305b7c979)